### ISSUE TITLE
Javascript popup window documentation in html and javascript files is difficult to read the name of the function

### ISSUE THREAD

**Issue by vt99** [Revivius/nb-darcula#108]
Possible to make a darker background.
![jsdoc_popup](https://cloud.githubusercontent.com/assets/10602367/15602492/2bfa5020-23fe-11e6-8352-882facbed1e3.png)

**Comment by Revivius** [comment]
@vt99; this is due to hardcoded background color in documentation text:

`
StringBuilder sb = new StringBuilder("<p style=\"background-color: #C7C7C7; width: 100%; padding: 3px; margin: 10 5 3 5;\">\n"); //NOI18N`

see: http://hg.netbeans.org/main/file/294dca090459/javascript2.editor/src/org/netbeans/modules/javascript2/editor/doc/JsDocumentationPrinter.java#l141


Since that string was not assigned to a field I dont know a good way of overriding that with reflection and in any case there should not be a hardcoded color so please report this in NetBeans bugzilla (may be with a link to this issue).

Thank you for reporting.

**Comment by markiewb** [comment]
See https://netbeans.org/bugzilla/show_bug.cgi?id=231808#c5

**Comment by Revivius** [comment]
@vt99 please vote for bug mentioned by @markiewb, I did.

**Comment by markiewb** [comment]
Fixed

https://netbeans.org/bugzilla/show_bug.cgi?id=231808#c8

**Comment by vt99** [comment]
Thanks you.
@Revivius tell me please how to vote for a bug?

**Comment by Revivius** [comment]
@markiewb can we use reflection to fix the colors now ?

**Comment by markiewb** [comment]
@Revivius: Yes, you should be able to access the fields in 8.2. But for future fixes - especially in NB code - I am not available, because I am not a NB user (or committer or plugin author) anymore.
